.travis.yml

```diff
@@ -16,21 +16,22 @@ install:
 - pip install --require-hashes -r requirements.txt
 - pip install -r requirements_for_test.txt
 before_script:
+- npm run compile
 - "./scripts/run_app.sh"
 script:
 - "./scripts/run_tests.sh"
 # fail travis if under X% test coverage
 - coverage report --fail-under=77
 notifications:
--  slack: digitaleq:LDxJXRGeVmg3PpdUhp90eY3Z
+- slack: digitaleq:LDxJXRGeVmg3PpdUhp90eY3Z
```

Comment:
Are the notifications working? travis docs suggests that it should notify every time https://docs.travis-ci.com/user/notifications/ but I haven't got a notification or I am not subscribed to the correct channel?